### DISCUSSION TITLE
Benchmark against quick_cache

### DISCUSSION THREAD

**Discussion by esemeniuc** [wvwwvwwv/scalable-concurrent-containers#166] [Q&A]
Any chance this quick_cache could be added to the testing roster? https://github.com/arthurprs/quick-cache?tab=readme-ov-file#benchmarks

**Comment by wvwwvwwv** [comment]
Interesting topic, but `quick-cache` provides much more features (global LRU, per-item weights, pinning, event hooks, etc.) than `scc::HashCache`, therefore this won't be a fair comparison.
